src/app/tide/spectrum_preprocess2.cc

```diff
@@ -76,8 +77,16 @@ void ObservedPeakSet::PreprocessSpectrum(const Spectrum& spectrum, int charge,
 
   max_mz_.InitBin(min(experimental_mass_cut_off, max_peak_mz));
   cache_end_ = MaxBin::Global().CacheBinEnd() * NUM_PEAK_TYPES;
-
   memset(peaks_, 0, sizeof(double) * MaxBin::Global().BackgroundBinEnd());
+  memset(raw_peaks_, 0, sizeof(double) * MaxBin::Global().BackgroundBinEnd());
```

Comment:
what is the difference between peaks and raw_peaks. Do they have to be different?